### PR TITLE
Task/add docker files

### PR DESCRIPTION
**Pull request by dmoranj**
Add Docker files to the root of the project to enable Docker Hub to upload the images.

### CODE REVIEW

File: documentation/deployment.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ The CEP is a standard Node.js app and doesn't require more dependencies than the
...
 A mongoDB 2.4 database should be working and accesible before perseo can be started
 
+### Installation using Docker
+
+The last development version is uploaded as a Docker image to Docker Hub for every PR merged into the `develop` branch.
+Perseo FE needs some components to be present event to be started. Those components can be configured using:
+
+* Environment variables, as in the following example:
+```
+docker run -e "PERSEO_MONGO_HOST=52.26.28.74" -e "PERSEO_CORE_URL=http://52.26.93.210:8080" fiwareiotplatform/perseocore
```
Comment (fgalan) [inline documentation/deployment.md]: Not sure if we should include public IPs in public documents (even when they belong to a very well known infrastructure provider, see http://whatmyip.co/view/ip_addresses/874126336/52.26.28.0_52.26.28.255)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  && yum install -y npm git 
+
+WORKDIR /opt
+RUN git clone https://github.com/telefonicaid/perseo-fe.git && cd perseo-fe && git checkout develop && npm install
+
+EXPOSE 9090
+WORKDIR /opt/perseo-fe
+
+ENV PERSEO_MONGO_HOST=mongodb
+ENV PERSEO_CORE_URL=http://corehost:8080
```
Comment (fgalan) [inline Dockerfile]: Who is `corehost` ?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
+FROM centos:6
+
+RUN yum update -y && yum install -y wget \
+  && wget http://ftp.rediris.es/mirror/fedora-epel/6/i386/epel-release-6-8.noarch.rpm && yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm \
```
Comment (fgalan) [inline Dockerfile]: You should chose a more "neutral" mirror. Not everybody downloading/using this is located in Spain ;)

File: Dockerfile (hunk 2 above)
Comment (dmoranj) [inline Dockerfile]: "corehost" is the name we are giving to the host where Perseo Core is located. Docker has a system to link applications that consists on manipulating the /etc/hosts file adding entries for other machines that want to be linked. In this case, when someone wants to link a Perseo Core Docker to this Perseo FE one, he has to do it, giving the Perseo Core Docker the alias `corehost`.

Comment (dmoranj) [inline Dockerfile]: You can see an example in the Doc, BTW.

Comment (fgalan) [inline Dockerfile]: Do you mean this example?

> Or links to other docker images running in the same docker host, as in the following example:

Comment (dmoranj) [inline Dockerfile]: Exactly that. Maybe I should explain more extensively the links and their names.

File: Dockerfile (hunk 3 above)
Comment (dmoranj) [inline Dockerfile]: FIxed in f043447

File: documentation/deployment.md (hunk 1 above)
Comment (dmoranj) [inline documentation/deployment.md]: Fixed in 8ec1a8b

File: Dockerfile (hunk 2 above)
Comment (fgalan) [inline Dockerfile]: I think the example should be more clear. For me at least (docker newbie :) is a bit obscure...

Comment (dmoranj) [inline Dockerfile]: I tried to improve the documentation in e175315

**Comment by fgalan** [comment]
LGTM

**Pull request merged**